### DISCUSSION TITLE
Sites load weirdly with point blank

### DISCUSSION THREAD

**Discussion by Synaptic149** [3kh0/ext-remover#408] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
I'm using the point blank exploit with the CAUB exploit to disable Lightspeed Filter (formerly known as Relay). However, what I've found is depending on the exact second I open a page on Discord, it's able to block images and videos. I also can't use certain sites like Snapchat for web, or Reddit.

I know that point blank works by toggling the "allow access to file URLs" setting 1 time every second. I'm wondering if there's a way I could make it go faster. I tried changing the number at the end of the javascript code but it didn't do anything.

**Comment by obkduishinijf** [comment, marked as answer]
using school wifi or home wifi

**Reply by Synaptic149** [reply]
I'm using a hotspot.
